### ISSUE TITLE
honeybrid configure question

### ISSUE THREAD

**Issue by zhanz5** [tklengyel/honeybrid#4]
If the IP of the low-interaction honeypot and the high-interaction honeypot is the local IP, it only takes up a different port. How should I set the configuration file, the IP of the two honeypots is different in the default configuration file. Thank you for your answer.

**Comment by tklengyel** [comment]
I'm not sure, it's been a while since I've setup honeybrid but I never used just one IP for all honeypots either. Using the system where honeybrid is running as your HIH is probably not the best idea.

**Comment by zhanz5** [comment]
> I'm not sure, it's been a while since I've setup honeybrid but I never used just one IP for all honeypots either. Using the system where honeybrid is running as your HIH is probably not the best idea.

Thank you, and I have another question. In your configuration file, the honeypot configuration is like this, 
![image](https://user-images.githubusercontent.com/37404833/50940557-57259a00-14bc-11e9-98bd-35e23c18206d.png)
but the configuration I saw at http://honeybrid.sourceforge.net/ is like this, 
![image](https://user-images.githubusercontent.com/37404833/50940698-ea5ecf80-14bc-11e9-806a-dfd3b6d40054.png)
 so it looks simpler, so I chose this one.  But directly report syntaxr error after running. Is it in the use of this version, only follow your target configuration

**Comment by tklengyel** [comment]
Yes, the latest version has a lot more options. If you want the simpler config you need to use the previous releases.

**Comment by zhanz5** [comment]
> Yes, the latest version has a lot more options. If you want the simpler config you need to use the previous releases.
Thank you for your answer. I am a bit confused about the instructions in the README. I am sorry that I have so many low-level questions.
![image](https://user-images.githubusercontent.com/37404833/50941223-ee8bec80-14be-11e9-9a88-479124522a2f.png)
In the figure, the address of the honeypot was changed while editing the configuration file. Why  still need bind honeybrid to my network and honeypots IP addresses?
